rskj-core/src/main/java/co/rsk/peg/constants/BridgeTestNetConstants.java

```diff
@@ -51,23 +48,14 @@ public class BridgeTestNetConstants extends BridgeConstants {
             Hex.decode("034844a99cd7028aa319476674cc381df006628be71bc5593b8b5fdb32bb42ef85")
         );
 
-        List<BtcECKey> genesisFederationPublicKeys = Arrays.asList(
+        genesisFederationPublicKeys = Arrays.asList(
             federator0PublicKey,
             federator1PublicKey,
             federator2PublicKey,
             federator3PublicKey
         );
 
-        // IMPORTANT: BTC, RSK and MST keys are the same.
-        // Change upon implementation of the <INSERT FORK NAME HERE> fork.
-        List<FederationMember> federationMembers = FederationMember.getFederationMembersFromKeys(genesisFederationPublicKeys);
-
-        // Currently set to:
-        // Currently set to: Monday, October 8, 2018 12:00:00 AM GMT-03:00
-        Instant genesisFederationAddressCreatedAt = Instant.ofEpochMilli(1538967600l);
-
-        FederationArgs federationArgs = new FederationArgs(federationMembers, genesisFederationAddressCreatedAt, 1L, getBtcParams());
-        genesisFederation = FederationFactory.buildStandardMultiSigFederation(federationArgs);
+        genesisFederationCreationTime = ZonedDateTime.parse("2018-10-08T03:00:00Z").toInstant();
```

Comment:
Is the instant object created from `ZonedDateTime` class the same as `Instant.ofEpochMilli(1538967600l)`?

Reply:
Yes, you can see that in `BridgeConstantsTest` class. In there, we make sure that this is the correct value.